dom/nodes/Document-createEvent.html

```diff
@@ -99,48 +101,56 @@
   "IDBVersionChangeEvent",
   "InstallEvent",
   "KeyEvent",
-  "MIDIConnectionEvent",
```

Comment:
Different defaults, great :) I guess whichever editor sorted lines first wins in each case.